src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '-':
+      return Big(numberOne) - Big(numberTwo);
+    case '+':
+      return Big(numberOne) + Big(numberTwo);
+    case '/':
+      return Big(numberOne) / Big(numberTwo);
+    case '%':
```

Comment:
You have to manage division by Zero on the live demo. You know that will cause an error. Take note of that on the next milestone

Reply:
Thanks, I'll definitely add this